packages/react/src/DiagnosticReportDisplay/DiagnosticReportDisplay.tsx

```diff
@@ -48,17 +50,32 @@ const useStyles = createStyles((theme) => ({
 export interface DiagnosticReportDisplayProps {
   value?: DiagnosticReport | Reference<DiagnosticReport>;
   hideObservationNotes?: boolean;
+  hideSpecimenInfo?: boolean;
 }
 
+DiagnosticReportDisplay.defaultProps = {
+  hideObservationNotes: false,
+  hideSpecimenInfo: false,
+} as DiagnosticReportDisplayProps;
+
 export function DiagnosticReportDisplay(props: DiagnosticReportDisplayProps): JSX.Element | null {
+  const medplum = useMedplum();
   const diagnosticReport = useResource(props.value);
-  const specimen = useResource(diagnosticReport?.specimen?.[0]);
+  const [specimens, setSpecimens] = useState<Specimen[]>();
+
+  useEffect(() => {
+    if (diagnosticReport?.specimen) {
+      Promise.all(diagnosticReport.specimen.map((ref) => medplum.readReference(ref)))
```

Comment:
Ok for now, but `Promise.all` will fail if any of the individual operations fail.